apps/protocol-api/src/prisma/resolvers/UserGuild.ts

```diff
@@ -54,19 +67,69 @@ export class GuildUserCustomResolver {
   ) {
     const address = req.session.siwe.data.address;
 
+    console.log('args', args);
+
     const user = await prisma.user.findFirst({
       where: { address: { equals: address } },
     });
 
-    if (user?.id !== args.data.userId) {
+    if (args.data.guildId === undefined && user?.id !== args.data.userId) {
```

Comment:
Is this logic correct?  Shouldn't this be guildId not undefined and context user id does not equal args userId?

Reply:
Yup, great catch. I think when reworking this the other day I reversed it.

Reply:
Oh. I realize now why it's not this way -- we're creating the DAO first and then adding the users to it, so by the time it gets to this point it has an ID. Let me try to rework this.

We had to create the DAO first and then link the users to it otherwise it was creating a new DAO for each user

Reply:
This should be fixed now